tools/capacity-planner-cli/README.md

```diff
@@ -0,0 +1,163 @@
+# Capacity Planner CLI
+
+This tool is a stand-alone tool to extract peak resource usage values and corresponding timestamps for a given GCP project, time range and timezone.
+
+
+## Usage
+
+You should run the following command to acquire the Application Default Credentials (ADC). Alternatively you may want to configure `GOOGLE_APPLICATION_CREDENTIALS` environment variable to use your service accounts to avoid hitting lower quota with ADC.
+
+```
+gcloud auth application-default login
+```
+
+```
+% pip install -r requirements.txt
+% python capacity_planner.py --help
+Usage: capacity_planner.py [OPTIONS]
+
+Options:
+  --project_id TEXT               GCP project ID where the Cloud Monitoring
+                                  API is called against.  [required]
+  --end_time [%Y-%m-%dT%H:%M:%S%z]
+                                  The end time in ISO 8601 format of the time
+                                  interval for which           results should
+                                  be returned. Default is now.           e.g.
+                                  2022-10-03T05:23:02+09:00
+  --duration_minutes INTEGER      The number of minutes in the time interval
```

Comment:
Are there any limits on the duration? e.g. maximum, would be good to specify if any

Reply:
Added help comments to indicate the maximum